master/getting-started/kubernetes/installation/calico.md

```diff
@@ -89,7 +89,10 @@ datastore type and number of nodes.
 
    We recommend at least one replica for every 200 nodes and no more than
    20 replicas. In production, we recommend a minimum of three replicas to reduce
-   the impact of rolling upgrades and failures.
+   the impact of rolling upgrades and failures.  The number of replicas should 
+   always be less than the number of nodes, otherwise rolling upgrade will stall
+   (and Typha only helps with scale if there are fewer Typha instances than 
+   there are nodes).
```

Comment:
Should it be rolling upgrade*s*? (line 93
I would suggest converting the parenthetical into a new sentence, so you can just add a period after "stall", delete the parentheses and "and." You could add "In addition,"